llama/runner/runner.go

```diff
@@ -116,6 +131,9 @@ type Server struct {
 	// context window size
 	numCtx int
 
+	// does this model require a beginning of sequence token?
+	bosToken int
```

Comment:
should this be a `bool`?

Reply:
Logically yes, though it gets added to the number of tokens, which is why I made it an int.